src/ProjectManager.h

```diff
@@ -48,6 +48,33 @@ class AUDACITY_DLL_API ProjectManager final
    //! False when proj is not null and it is unsafe to overwrite it with contents of an .aup3 file
    static bool SafeToOpenProject(AudacityProject *proj);
 
+   //! Callable object that supplies the `chooser` argument of ProjectFileManager::OpenFile
```

Comment:
You give an explanation for this class in the commit comment, but not in the code.  Its purpose should be explained here.  What does it choose?  Why?

Reply:
ok